enhancements/baremetal-instance-types/README.md

```
@@ -0,0 +1,749 @@
+    rdma: "true"
+status:
+  phase: Available
+  conditions:
+    - type: HostsAvailable
+      status: "True"
+      reason: HostsAvailable
+      lastTransitionTime: "2026-06-15T12:00:00Z"
+```
+
+**`oc get bmit` output:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)